docs/docs/end-users/Configuration.fsx

```diff
@@ -573,17 +572,41 @@ formatCode
 (*** include-it ***)
 
 (**
-## G-Research style
+<fantomas-setting name="fsharp_multiline_bracket_style" green orange gr></fantomas-setting>
```

Comment:
I would remove the `orange` here. All settings will soon(ish) be in the style guide.